library/src/test/java/com/datatorrent/lib/db/jdbc/JdbcOperatorTest.java

```diff
@@ -375,14 +411,37 @@ public void testJdbcPojoInputOperator()
     OperatorContextTestHelper.TestIdOperatorContext context = new OperatorContextTestHelper.TestIdOperatorContext(
         OPERATOR_ID, attributeMap);
 
-    insertEventsInTable(10);
```

Comment:
Could you please make another function to add time based entries? This can be used at other places too.